Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -89,6 +89,47 @@ def query(self, section: str, argument: str = None, sub_section: str = 'general'
 ''' HELPER FUNCTIONS '''
 
 
+def build_context_indicator_no_results_status(indicator: str, indicator_type: str,
+                                              integration_name: str, reliability: Any = None) -> CommandResults:
+    indicator_map = {
+        'file': DBotScoreType.FILE,
+        'ip': DBotScoreType.IP,
+        'domain': DBotScoreType.DOMAIN,
+        'url': DBotScoreType.URL
+    }
```

Comment:
Why do you need the `indicator` map? why not pass this value instead of `indicator_type`?
Seems like this can be solved using `DBotScoreType`.

The same goes for other implementations of this function.